README.md

```diff
@@ -1,6 +1,8 @@
 # properties-file-image-relocation
 Image relocation for Java properties files.
 
+> **NOTICE** As of October 31, 2021, any use of this project is deprecated and we recommend using [Carvel imgpkg](https://carvel.dev/imgpkg/) instead.
+
```

Comment:
Fair enough. Do you want to archive this project too to make it clear there will be no further updates?

Reply:
I think that would be best. I'm updating the SCDF for K8s docs to use `imgpkg` and I know of no other users for this project.